modules/aws_account_permissions/variables.tf

```
@@ -0,0 +1,32 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# Required PARAMETERS
+# These variables have default values and can be modified based on the requirement.
+# ---------------------------------------------------------------------------------------------------------------------
+
+
+variable "iam_role_name" {
+  description = "Customize the name of IAM role for Illumio AWS integration"
+  type        = string
+  default     = "IllumioCloudIntegrationRole"
+  validation {
+    condition     = length(var.iam_role_name) > 1
+    error_message = "The IAM role name cannot be empty."
+  }
+}
+
+variable "access_mode" {
+  description = "integration mode Read/ReadWrite."
+  type        = string
+  default     = "ReadWrite"
+  validation {
+    condition     = contains(["Read", "ReadWrite"], var.access_mode)
+    error_message = "Must be Read or ReadWrite"
+  }
+}
+
```

Add a variable for the account name in CloudSecure:
```suggestion

variable "name" {
  description = "The name of this account in CloudSecure."
  type        = string
  validation {
    condition     = length(var.name) > 1
    error_message = "The name value must not be empty."
  }
}
```